DuckDuckGo.xcodeproj/xcshareddata/xcschemes/AppScreenshots.xcscheme

```diff
@@ -1,103 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why is this removed?

Reply:
We don't generate screenshots via the screenshot now. I already removed the associated code and target but forgot to remove this scheme, so just a clean up.